mods/player_api/api.lua

```diff
@@ -62,6 +64,15 @@ function player_api.set_model(player, model_name)
 	player_model[name] = model_name
 end
 
+function player_api.reset_boxes(player, model_name)
+	local model = models[model_name]
```

Comment:
Why do you need `model_name` after all? It would be much more usable to restore the collision box of the currently active model, i.e.:
`local model = player_model[player:get_player_name()]`

Reply:
Yes will do. Something obvious i missed.